README.md

```diff
@@ -24,10 +24,30 @@ represents one thing that your application *does*.
 
 ### Context
 
-An interactor is given a *context*. The context contains everything the
-interactor needs to do its work.
+An interactor is given a *context*.  The context contains everything the
+interactor needs to do it's work.
 
-When an interactor does its single purpose, it affects its given context.
+The context is created when the interactor is called with a hash:
+
+In the example below the hash { email: 'example@example.com'} is passed 
+and is turned into OpenStruct object and becomes the context.
```

Comment:
The context object IMHO, appears magical as I don't immediately see the transformation from named parameters or a hash into  Interactor::Context.  I thought the context object at first was some sort of Hashie ( https://github.com/intridea/hashie ) object, but I wasn't sure how to go about assigning new values to it?  Do I declare them?  Are they auto-magically created?, thus the reference to openstruct so that people understand how to work with the context